ibis/mapd/client.py

```diff
@@ -101,6 +101,29 @@ class MapDDataType:
         'MULTIPOLYGON': 'multipolygon',
     }
 
+    _TDatumType_enum = {
```

Comment:
maybe it could take a look in this line added some days ago https://github.com/ibis-project/ibis/blob/master/ibis/mapd/client.py#L22

Reply:
To me it seems better to include the enumeration explicitly, give that there are already mapd dependent type encodings in this file.  Why include a dependency on sys.version that could cause an unnecessary exception.  Of course I could be wrong :).

Reply:
sql_validate return  col_type=TTypeInfo(type=2 .... <-  This integer references type TDatumType string in _TDatumType enum.  Is there a way to reference pymapd.dtypes.TDatumType using this same integer, because it appears from my inspection that the enumeration is different.